### PR TITLE
Update rpi4_reference_dunfell.json

### PR DESCRIPTION
**Pull request by sravanthi298**
updated rpi4_reference_dunfell.json for DAC-cobalt

### CODE REVIEW

File: templates/generic/rpi4_reference_dunfell.json [hunk 1]
```diff
@@ -94,12 +94,27 @@
                     "rw",
                     "X-dobby.optional"
                 ]
+            },
```
Comment (sverkoye) [inline templates/generic/rpi4_reference_dunfell.json:97]: This binds the whole /tmp from the host. That is too much. For the rialto socket add this:

            {
                "source": "/tmp/{id}",
                "destination": "/tmp/rialto-0",
                "type": "bind",
                "options": [
                    "rbind",
                    "rw",
                    "X-dobby.optional"
                ]
            }

Same as I do here: https://github.com/rdkcentral/BundleGen/blob/c6c2922d4889643ae69ada992c19126e310ecec7/templates/generic/rpi3_reference_dunfell.json#L89

File: templates/generic/rpi4_reference_dunfell.json [hunk 2]
```diff
@@ -94,12 +94,27 @@
...
+                "nodev",
+                "rw"
+            ],
+            "type": "bind"
             }
         ],
         "envvar": [
             "LD_PRELOAD=/usr/lib/libGLESv2.so.2:/usr/lib/libwayland-egl.so.1",
             "XDG_RUNTIME_DIR=/tmp",
-            "RIALTO_SOCKET_PATH=/tmp/rialto-0"
```
Comment (sverkoye) [inline templates/generic/rpi4_reference_dunfell.json:102]: This must stay /tmp/rialto-0 if you mount bind the rialto socket as I mentioned above.

File: templates/generic/rpi4_reference_dunfell.json [hunk 3]
```diff
@@ -94,12 +94,27 @@
...
+            "RIALTO_SOCKET_PATH=/tmp/{id}",       
+            "SAGEBIN_PATH=/usr/bin/",
+            "LD_LIBRARY_PATH=/usr/lib:/usr/lib/wpeframework/plugins",
```
Comment (sverkoye) [inline templates/generic/rpi4_reference_dunfell.json:115]: Are you mounting framework plugins under this container? If not, then adding that path is useless and only confusing.

**Review by sverkoye — CHANGES_REQUESTED** [review]

**Comment by sravanthi298** [comment]
With latest Rpi4 OCI bundle, DAC-cobalt is working fine.

 These changes are not required now. Hence closing this PR.

**Pull request closed without merging**